### ISSUE TITLE
Could not authenticate to gateway (No cookie given)

### ISSUE THREAD

**Issue by sebastianbauer** [theinvisible/openfortigui#8]
Just installed 0.3.0 to test the 2FA feature. After clicking connect I get the following log. The VPN entry was deleted and re-created manually. The gateway works with `openfortivpn` on CLI.

```
INFO:   Connected to gateway.
ERROR:  Could not authenticate to gateway (No cookie given).
INFO:   Closed connection to gateway.
INFO:   Logged out.
```

**Comment by theinvisible** [comment]
Can you post the working log output of openfortivpn please? Regarding you log there is never a 2FA happen. Thanks.

**Comment by sebastianbauer** [comment]
I just closed the app via the tray icon (but it actually was a fresh install)... Now it works...

**Comment by sebastianbauer** [comment]
But setting the DNS (option is enabled) doesn't work :-(

**Comment by sebastianbauer** [comment]
```
INFO:   Removing VPN nameservers...
INFO:   Connected to gateway.
2factor authentication token: 
INFO:   Authenticated.
INFO:   Remote gateway has allocated a VPN.
INFO:   Custom: fg.tecxipio.com.
INFO:   Got addresses: [x.x.x.x], ns [0.0.0.0, 0.0.0.0]
INFO:   Interface ppp0 is UP.
INFO:   Setting new routes...
INFO:   Adding VPN nameservers...
INFO:   Tunnel is up and running.
INFO:   Setting ppp interface down.
INFO:   Restoring routes...
INFO:   Removing VPN nameservers...
```

**Comment by theinvisible** [comment]
Okay i must validate this. So you  can see the right DNS-servers in log but they are not applied to system right?

**Comment by sebastianbauer** [comment]
Sorry for missing explanation. I actually get 0.0.0.0 in the log what is not the DNS address. Running `openfortivpn` on CLI works and gets the correct DNS (also in the output).

**Comment by theinvisible** [comment]
Hello, this is fixed now in the latest release (0.3.3.). Give it a try. : )
